go_modules/spec/dependabot/go_modules/file_updater/go_mod_updater_spec.rb

```diff
@@ -236,12 +236,23 @@
             # OpenAPIV2 has been renamed to openapiv2 in this version
             let(:dependency_version) { "v0.5.1" }
 
-            it "raises a DependencyFileNotResolvable error" do
-              error_class = Dependabot::DependencyFileNotResolvable
+            # NOTE: We explicitly don't want to raise a resolvability error from `go mod tidy`
+            it "does not raises a DependencyFileNotResolvable error" do
               expect { updater.updated_go_sum_content }.
-                to raise_error(error_class) do |error|
-                expect(error.message).to include("googleapis/gnostic/OpenAPIv2")
-              end
+                to_not raise_error
+            end
+
+            it "updates the go.mod" do
+              # this is failing, but I'm not sure why.
+              # The code runs `go get -d github.com/googleapis/gnostic@v0.5.1`
+              # and then later runs `go mod tidy -e`.
+              # So I manually ran those in the test fixture, and it resulted in
+              # this line appearing. But when the test executes, this line is missing.
+              # I'm not sure why, and not sure how to run the Ruby debugger
+              # to step through it.
```

Comment:
The test was failing, and after looking into it, I understand why. So I replicated the steps manually, and this is what I expect.

But the test is still failing, so at this point unclear if the test is just plain wrong, or if there's some wrong code, or what.

Anyone else want to take a stab at it?